modules/components/src/AdvancedFacetView/utils.js

```diff
@@ -97,11 +97,19 @@ const filterDisplayTreeDataBySearchTerm = ({
 }) => {
   const shouldBeIncluded = ({ title, path, children }) => {
     const inTitle = title.match(strToReg(searchTerm));
+    if (inTitle) {
```

Comment:
less compact but it avoids computing expensive stuff unnecessarily and it's really less readable in my option.